context/clientEntries.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const { fromPairs } = require('lodash');
+
+const { getPathFromCwd } = require('../lib/cwd');
+
+const defaultClientEntry = 'skuDefaultClientEntry';
```

Comment:
Could this be `main`? `skuDefaultClientEntry` looks kinda weird in the generated file names.